tests/rest/client/v1/test_rooms.py

```diff
@@ -688,7 +688,7 @@ class RoomJoinRatelimitTestCase(RoomBase):
     )
     def test_join_local_ratelimit(self):
         """Tests that local joins are actually rate-limited."""
-        for i in range(5):
+        for i in range(3):
```

Comment:
It is unclear to me whether this number is important or if it just eventually needs to be rate-limited.

Reply:
This is the maximum number of requests you can send before the it gets rate-limited. I'm actually surprised it's passing, iirc that config lets slightly more than 3 requests through (5 as far as I could tell).

Reply:
So we figured out where that was coming from (and how to fix it):

![image](https://user-images.githubusercontent.com/5547783/91295150-8ae92c80-e792-11ea-8045-4898babc04e4.png)